.. title: Research_
.. slug: research
.. date: 2018-11-11 17:13:05 UTC-04:00
.. tags:
.. link:
.. description:
.. type: text

I am currently focused on the evolution of repetitive DNA sequences, both transposable elements--mobile genetic parasites--and satellite DNA, long stretches of DNA where the same sequence is repeatd over and over again in tandem. These often comprise large portions of eukaryotic genomes and are subject to interesting and poorly understood modes of evolution. Their repetitive nature means that they confuse the recombination machinery, which can shape both sequence and copy number in ways that happen for single-copy sequence. Both classes of sequence are associated with repressive chromatin states, and so they both have the potential to influence gene expression and underlie genetic diseases (eg contraction of a human satellite in the subtelomere causes fascioscapulohumeral muscular dystrophy). Finally transposable elements are absolutely in conflict with the rest of the genome, and some satellites are engaged in selfish evolution as well. 

But there are technical challenges that have hindered our ability to study these classes of sequence--the tools that work well for single copy sequences, don't often work well on repeats. Part of my PhD has been spent developing tools to circumvent these challenges and use available data to gain insight into their evolution.

The evolutionary questions that excite me the most are those the get down to the fundamental processes. There are rates by which repeat copy number and sequence evolve and different mutational processes that could underlie these rates--I want details about what these are. The expansion or contraction of a particular satellite array might have no impact on an organism's fitness or it might have phenotypic consequences. In what ways would these scenarios shape satellite variation? Does real variation look more like one or the other? How do new satellites arise?

 