<!DOCTYPE html>
<html lang="english">

<head>
  <!-- ## for client-side less
  <link rel="stylesheet/less" type="text/css" href="https://laptopbiologist.github.io/theme/css/style.less">
  <script src="http://cdnjs.cloudflare.com/ajax/libs/less.js/1.7.3/less.min.js" type="text/javascript"></script>
  -->
  <link rel="stylesheet" type="text/css" href="https://laptopbiologist.github.io/theme/css/style.css">
  <link rel="stylesheet" type="text/css" href="https://laptopbiologist.github.io/theme/css/pygments.css">
  <link rel="stylesheet" type="text/css" href="//fonts.googleapis.com/css?family=PT+Sans|PT+Serif|PT+Mono">

  <meta charset="utf-8" />
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="author" content="Michael McGurk">
  <meta name="description" content="Posts and writings by Michael McGurk">

  <link href="https://laptopbiologist.github.io/feeds/all.atom.xml" type="application/atom+xml" rel="alternate" title="Laptop Biology - Mike McGurk Atom" />


  <title>
    Laptop Biology - Mike McGurk
&ndash; Research  </title>

</head>

<body>
  <aside>
    <div id="user_meta">
      <a href="https://laptopbiologist.github.io">
        <img src="https://laptopbiologist.github.io/theme/images/logo.png" alt="logo">
      </a>
      <h2><a href="https://laptopbiologist.github.io">Michael McGurk</a></h2>
      <p></p>
      <ul>
        <li><a href="https://laptopbiologist.github.io/pages/curriculum-vitae.html">Curriculum Vitae</a></li>
        <li><a href="https://laptopbiologist.github.io/pages/about.html">About</a></li>
        <li><a href="https://laptopbiologist.github.io/pages/publications.html">Publications</a></li>
        <li><a href="https://laptopbiologist.github.io/pages/research.html">Research</a></li>
        <li><a href="http://twitter.com/laptopbiologist" target="_blank">Twitter</a></li>
        <li><a href="http://github.com/laptopbiologist" target="_blank">Github</a></li>
        <li><a href="mailto:mpm289@cornell.edu" target="_blank">Email</a></li>
      </ul>
    </div>
  </aside>

  <main>
    <header>
      <p>
      <a href="https://laptopbiologist.github.io">Index</a> &brvbar; <a href="https://laptopbiologist.github.io/archives.html">Archives</a>
      &brvbar; <a href="https://laptopbiologist.github.io/feeds/all.atom.xml">Atom</a>
      </p>
    </header>

<article>
  <div class="article_title">
    <h3><a href="https://laptopbiologist.github.io/pages/research.html">Research</a></h3>
  </div>
  <div class="article_text">
    <h4>Current</h4>
<p>My current research focus is the within species evolution of highly repeated DNA sequence, both TEs and satellite DNA. Transposable elements are genetic parasites which can make copies of themselves at new locations dispersed throughout a genome. They are ubiquitious and can comprise large proportions of eukaryotic genomes, often outnumbering functional genes. Satellite DNA, on the other hand, are regions of the genome where the same sequence is repeated over and over again in tandem, for thousands or millions of basepairs. Satellites are not implicitly mobile and tend to be found in subtelomeric and pericentric sequence. </p>
<p>While these two classes of sequence appear disparate at first glance, there are both technical and biological factors that unify their study. First, repetitive sequence is challenging to analyze with the most common types of sequencing data (Illumina short-reads) using standard tools. A consequence is that we understand far less about the evolution and impacts of TE and satellite variation than we do for single-copy sequence. So, any tool that could circumvent the general challenges of interpreting repeat-derived sequencing reads would provide insights into both classes of sequence. The first task of my dissertation research was to develop a general tool for leveraging population genomic dataset to study repeats, <a href="https://doi.org/10.1101/158386">ConTExt</a>, which ultimately made it possible to ask a variety of questions about both satellites and TEs. From the biological angle, both classes of sequence are major components of constitutive heterochromatin, regions of the genome that are transcriptionally and recombinationally repressed. Much of this sequence is certainly nonfunctional junk DNA, but there is clear evidence that variation in heterochromatin can underlie reproductive barriers between related species, disrupt normal chromosome segregation in mitosis and meiosis, and alter gene expression locally and possibly globally (presumably by titrating chromatin binding factors). Further, some surprising connections between TEs and satellite DNA have come out of this project. Most strikingly, the fundamental characteristic of TEs--transposition-- regularly generates substrate from which new satellite arrays can emerge.</p>
  </div>
  <div class="article_meta">
    <p>Updated on: Mon 15 October 2018</p>
  </div>
</article>


    <div id="ending_message">
      <p>&copy; Michael McGurk. Built using <a href="http://getpelican.com" target="_blank">Pelican</a>. Theme by Giulio Fidente on <a href="https://github.com/gfidente/pelican-svbhack" target="_blank">github</a>. </p>
    </div>
  </main>
</body>
</html>